README.md

```diff
@@ -204,16 +211,24 @@ Section 3.1 with the following exceptions:
   MUST publish on `Ga` a sequence of `group/add-members` messages according to
   [ssb-meta-feeds-group-spec] Section 3.1, such that the union of all recipients
   in that sequence equals all remaining members
-
-It is RECOMMENDED that epoch `G` is the "most preferred epoch" among all the
-epochs that `a` is a member of, which succeed a certain epoch zero.
+>   - TODO what is the `tangles.group` + `tangles.member` for these `group/add-member`
+>   messages. I think it should be
+>     - `tangles.group`:
+>       - root: the epoch zero group/init message id
+>       - previous: latest message
+>     - `tangles.member`:
+>       - root: the group H group/init message id !?
+>       - previous: the previous `group/add-member` message for group H (if it exists)
+    
+It is RECOMMENDED that epoch `H` is the "most preferred epoch" among all the
+groups epochs that `a` is a member of.
 
 
 ### 4.2. Preferring the next epoch
 
-When a member `a` of an epoch `G` replicates and decrypts a `group/add-member`
-message that adds `a` to the new epoch `H` (in other words, `a` "detected the
-existence" of `H`), then `a` MUST select epoch `H` as "preferred" over `G`.
+When a member `b` of an epoch `G` replicates and decrypts a `group/add-member`
+message that adds them to the new epoch `H` (in other words, `b` "detected the
+existence" of `H`), then `b` MUST select epoch `H` as "preferred" over `G`.
```

Comment:
Hmm, this is an okay rename. I see what you're aiming for.

Reply:
yes, a cohesive narrative across parts

Reply:
done in another PR